client/state/selectors/is-automated-transfer-failed.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import {
+	flowRight as compose,
+} from 'lodash';
+
+/**
+ * Internal Dependencies
+ */
+import { transferStates } from 'state/automated-transfer/constants';
+import { getAutomatedTransferStatus } from 'state/automated-transfer/selectors';
+
+/**
+ * Maps automated transfer status value to indication if transfer is failed
+ *
+ * @param {String} status name of current state in automated transfer
+ * @returns {?boolean} is transfer currently failed? null if unknown
+ */
+export const isFailed = status =>
+	status
+		? status === transferStates.CONFLICTS || status === transferStates.FAILURE
+		: null;
```

Comment:
fine any way. you can probably blame me for inconsistencies. `null` can be useful for that tri-state value of "don't know" - just checking